keps/sig-network/563-dual-stack/README.md

```diff
@@ -1392,26 +1392,35 @@ This capability will move to stable when the following criteria have been met.
 
 * **Does enabling the feature change any default behavior?**
   Pods and Services will remain single-stack until cli flags have been modified
-  as described in this KEP. Once modified, existing and new services will remain
-  single-stack until user requests otherwise. Pods will become dual-stack
-  however will maintain the same ipfamily used before enabling feature flag.
+  as described in this KEP. Existing and new services will remain single-stack
+  until user requests otherwise. Pods will become dual-stack once CNI is
+  configured for dual-stack.
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes. If you decide to turn off dual-stack after turning on, ensure all
-  services are converted to single-stack first (switch ipfamily to single-stack
-  on all services) and then disable the feature. Remove the CLI parameters, as
-  an older client won't see or be able to use the new fields. When the user
-  disables dual-stack from the controller manager, new endpoints will no longer
-  carry two sets, while existing endpoints may not be updated.
+  Yes. If you decide to turn off dual-stack after turning on:
+    1. Ensure all services are converted to single-stack first (downgraded to
+       single-stack as described in this KEP)
+    2. Remove the CLI parameters.
+    3. Disable the feature.
```

Comment:
Are these two discrete steps?

Reply:
Yes; Kal stipulated that they should be in that order.